saleor/order/models.py

```diff
@@ -500,3 +500,10 @@ class Meta:
 
     def __repr__(self):
         return f"{self.__class__.__name__}(type={self.type!r}, user={self.user!r})"
+
+
+class Invoice(models.Model):
+    order = models.ForeignKey(Order, null=True, on_delete=models.SET_NULL)
+    number = models.CharField(max_length=64)
+    created = models.DateTimeField(auto_now_add=True)
+    url = models.URLField(max_length=256)
```

Comment:
Are you sure that 256 is enough in any cases?

Reply:
https://stackoverflow.com/a/417184

Reply:
such a long url's are a common effect of bad design but since we don't have impact on what will be integrated with this, we can go with 2000 limit for safety. Though I cannot imagine URL such long unless you pass everything in query params which is a horrible idea ;0

Reply:
I would stick with `255` which we use as a max value for other URL fields (e.g. webhook URLs). We control our integrations and if one of them produces longer URLs than this could be changed.

Reply:
We should support at least 2048 characters, we have no control over what external vendors want to put in their URLs. Look at what Facebook does to their image and sharing URLs for an example of what's out there.